sites/main-site/src/content/blog/2025/maintainers-minutes-2025-04-24.mdx

```diff
@@ -0,0 +1,146 @@
+James' concern was that the list of 'unassigned' modules can sometimes be very useful for training on writing modules or for hackathons where novice participants don't have something specific to work on (or want to practice before working on a specific pipeline).
+Removing or 'hiding' these issues would mean these would potentially hinder teachers or hackathon newcomers effectively begin their contributions.
+
+Eventually we found a compromise via a 🤖 'stale bot' concept that was assigned to Famke to implement.
+New-module issues that are not immediately assigned to work on by someone will be automatically given a 'wishlist' label.
+If it remains unassigned after a period time (initially around a year to ensure we always have a list of 'free' modules for hackathons), the issue wil be closed by an automated bot.
+The time period would reset if assigned to someone and then subsequently unassigned (if they could not complete it).
+
+Through this automated mechanism we hope it will make the nf-core/modules (our most popular repository!) more manageable for the community and the maintainers team, freeing up time to work on cooler things (or argue about naming things..., the maintainer team's favourite pasttime 😉).
+
+## Saving more polar bears via optimised AWS megatest configs
+
+<Image
+    src={notifications}
+    class="d-block m-auto"
+    width={400}
+    density={[1.5, 2]}
+    alt="Photo of a polar bear face with the text 'Save a polarbear, optimise your AWS megatests' overlaid"
+/>
+
+Finally, we briefly looked at a neat [little repository](https://github.com/FloWuenne/megatest-resource-optimization/tree/main/optimized_configs) that Florian (who unfortunately sent his apologies) has made that includes optimised resource-usage configuration files for nf-core pipelines AWS megatests.
+
+This was after observations that many pipeline's megatest configurations could be rather wasteful in over- or under-requesting resources for AWS - resulting in wasted costs and energy by using larger spot instances than necessary or causing unnecessary retries (sad 🐻‍❄️).
+
+We discussed whether there should be an automation to generate an optimised config and updating a pipeline's `test_full` config with the optimised config, or some other mechanism to incorporate these into our pipelines.
+
+We decided to ask volunteers to implement Florian's existing optimised configs to see if they cause any problems on a subset of nf-core pipelines before rolling out, and I discuss further on Slack (in fact, the Slack discussions were already fruitful and @Maxime and @Florian have already come up with a potential non-invasive system without requiring any extra work by pipeline developers - watch this space!)
```

Comment:
```suggestion
We decided to ask volunteers to implement Florian's existing optimised configs to see if they cause any problems on a subset of nf-core pipelines before rolling out. The Slack discussions were already fruitful. @Maxime and @Florian already came up with a potential non-invasive system without requiring any extra work by pipeline developers - watch this space!
```

Reply:
I didnt see the slack discussion, but add yourself as well if you were part of it @jfy133. I didn't quite if you were also involved in the discussion

Reply:
The discussion was here: https://nfcore.slack.com/archives/C043UU89KKQ/p1745593967346769?thread_ts=1745246825.260859&cid=C043UU89KKQ

I basically just said 'HELL YEAH' to the proposal from Maxime and Florian, so I don't think I'm necessary here